echopype/__init__.py

```diff
@@ -18,10 +18,12 @@
     "open_raw",
     "open_converted",
     "combine_echodata",
+    "verbose",
     "calibrate",
+    "clean",
+    "commongrid",
```

Comment:
I have similar question about the order of these too like in https://github.com/OSOceanAcoustics/echopype/pull/993/files#r1142620782

Reply:
In this case, I followed the pattern that seemed to be in place: functions first, then subpackages. I moved `verbose` to follow that pattern more strictly (but `combine_echodata` doesn't follow that order ...). But we could just follow a strict alphabetical order. Any preference? The order has no user-visible impact, so it's more about code readability.

Reply:
Let's do alphabetical order then? That way it's easier to find and update things if needed in the future.

Reply:
Done.